python/coral_usb/util.py

```diff
@@ -30,6 +30,33 @@ def get_panorama_sliced_image(panorama_img, panorama_slice):
     return img
 
 
+def get_tiles(width, height, overlap=True):
+    tile_sizes = [[300, 300], [250, 250]]
```

Comment:
```suggestion
def get_tiles(width, height, overlap=True, tile_sizes=[[300, 300], [250, 250]]):
```

can you set `tile_sizes` as  optional arguments?